test/parse-error-test/annotation_semantics/parse-error.expected

```diff
@@ -0,0 +1,4 @@
+[error] test/parse-error-test/annotation_semantics/input.ddl[3.4] failure: string matching regex `\p{javaJavaIdentifierStart}\p{javaJavaIdentifierPart}*' expected but `)' found
```

Comment:
<http://www.scala-lang.org/api/2.10.2/index.html#scala.util.parsing.combinator.Parsers$Parser>

You may need to use `~!` maybe?